### ISSUE TITLE
Update to tzdata 2022g

### ISSUE THREAD

**Issue by albertyw** [moment/moment-timezone#1018]
### Moment-timezone version which you use:

Version: 2022f

### Issue description: 

tzdata 2022g was released today with immediate updates to parts of Mexico.  I tried following the instructions in https://github.com/albertyw/moment-timezone/blob/develop/contributing.md but they don't result in the correct files being generated (for example, I get an exception when `abbr` returns `undefined` in https://github.com/moment/moment-timezone/blob/develop/moment-timezone.js#L266 amongst other issues in #1019)

Can we get moment-timezone updated to 2022g?  cc @lhorie

Changelog for 2022g:
```
Release 2022g - 2022-11-29 08:58:31 -0800

  Briefly:
    The northern edge of Chihuahua changes to US timekeeping.
    Much of Greenland stops changing clocks after March 2023.
    Fix some pre-1996 timestamps in northern Canada.
    C89 is now deprecated; please use C99 or later.
    Portability fixes for AIX, libintl, MS-Windows, musl, z/OS
    In C code, use more C23 features if available.
    C23 timegm now supported by default
    Fixes for unlikely integer overflows

  Changes to future timestamps

    In the Mexican state of Chihuahua, the border strip near the US
    will change to agree with nearby US locations on 2022-11-30.
    The strip's western part, represented by Ciudad Juárez, switches
    from -06 all year to -07/-06 with US DST rules, like El Paso, TX.
    The eastern part, represented by Ojinaga, will observe US DST next
    year, like Presidio, TX.  (Thanks to Heitor David Pinto.)
    A new Zone America/Ciudad_Juarez splits from America/Ojinaga.

    Much of Greenland, represented by America/Nuuk, stops observing
    winter time after March 2023, so its daylight saving time becomes
    standard time.  (Thanks to Jonas Nyrup and Jürgen Appel.)

  Changes to past timestamps

    Changes for pre-1996 northern Canada (thanks to Chris Walton):

      Merge America/Iqaluit and America/Pangnirtung into the former,
      with a backward compatibility link for the latter name.
      There is no good evidence the two locations differ since 1970.
      This change affects pre-1996 America/Pangnirtung timestamps.

      Cambridge Bay, Inuvik, Iqaluit, Rankin Inlet, Resolute and
      Yellowknife did not observe DST in 1965, and did observe DST
      from 1972 through 1979.

      Whitehorse moved from -09 to -08 on 1966-02-27, not 1967-05-28.

    Colombia's 1993 fallback was 02-06 24:00, not 04-04 00:00.
    (Thanks to Alois Treindl.)

    Singapore's 1981-12-31 change was at 16:00 UTC (23:30 local time),
    not 24:00 local time.  (Thanks to Geoff Clare via Robert Elz.)
```

**Comment by davidseek** [comment]
Kindly bumping. Can someone please look into the open pull requests ☺️

**Comment by gilmoreorless** [comment]
This has been released in version `0.5.40`.
